modules/apps/headless-apio/forms/forms-apio-impl/src/main/java/com/liferay/forms/apio/internal/resource/FormInstanceNestedCollectionResource.java

```diff
@@ -110,6 +114,36 @@ public String getName() {
 		).addLinkedModel(
 			"structure", StructureIdentifier.class,
 			DDMFormInstance::getStructureId
+		).addNested(
+			"settings", this::_getSettings,
+			nestedBuilder -> nestedBuilder.nestedTypes(
+				"FormInstanceSettings"
+			).addString(
+				"emailFromAddress", DDMFormInstanceSettings::emailFromAddress
```

Comment:
Have you checked schema.org?

Reply:
I'm not sure how can I adapt that to schema models, maybe ContactPoint model can fit the email properties.

Reply:
Ok, later! :)